src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeAccessServiceTest.java

```diff
@@ -59,4 +62,18 @@ public void assertCanUpdate_should_not_throw_an_exception_if_service_can_update_
         assertThatCode(() -> service.assertCanUpdate("jur_A", "update_A"))
             .doesNotThrowAnyException();
     }
+
+    @Test
+    public void getReadJurisdictionForService_should_return_name_of_then_jurisdiction_from_which_service_can_read() {
```

Comment:
Since when we started/decided to prefix with method name?

Reply:
See other existing tests in this file.
Also good imho, as there are now 2 methods in this service, so it's clear which tests are for which method.
Alternatively I could create a separate class.